src/pivot-table/components/cells/DataCell.tsx

```diff
@@ -52,13 +59,23 @@ const MeasureCell = ({ columnIndex, rowIndex, style, data }: MeasureCellProps):
   const isNumeric = isNull ? !Number.isNaN(+text) : true;
   const cellStyle = {
     ...(isNull ? nilStyle : numericStyle),
+    display: "flex",
     justifyContent: isNumeric ? "flex-end" : "center",
   };
 
   return (
     <div title={text} style={{ ...style, ...containerStyle }} data-testid={testId}>
-      <div style={{ ...cellStyle, display: "flex" }}>
-        <span style={{ ...textStyle, ...styleService.content, alignSelf: "flex-start" }}>{text}</span>
+      <div style={cellStyle}>
+        <span
+          style={{
+            ...textStyle,
+            ...styleService.content,
+            ...(!includesDigit(text) && getGridTextClampStyle(styleService.lineClamp)),
```

Comment:
We should use line clamp if the value is not numeric. So this should do it:
```suggestion
            ...(!isNumeric && getGridTextClampStyle(styleService.lineClamp)),
```

Reply:
it actually does not work as what we discussed, if we have number in text -> it does not limit lines back to only 1:

https://user-images.githubusercontent.com/29652890/233991850-72c87d43-2b73-440a-bd55-c1a32b2cb34a.mov

Reply:
I don't recall use saying that if it's contains it a number we should not line break it.

We want it to behave like in the sn-table. So if the cell value is a numeric value, no line break is allow. If the values is not a numeric, line break is allowed. Which should be what my suggested code does.